components/storefront/product-details.tsx

```
@@ -56,20 +57,16 @@ export function ProductDetails({ description, attributes }: ProductDetailsProps)
+  if (!html) return null;
   return (
-    <div className="max-w-prose space-y-3">
-      {paragraphs.map((paragraph, i) => (
-        <p key={i} className="text-sm leading-relaxed text-muted-foreground">
-          {paragraph}
-        </p>
-      ))}
-    </div>
+    <div
+      className="prose prose-sm max-w-prose dark:prose-invert"
+      dangerouslySetInnerHTML={{ __html: html }}
```

Biome (2.4.6)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)